Package.swift

```diff
@@ -0,0 +1,5 @@
+import PackageDescription
+
+let package = Package(
+    name: "ReSwift"
+)
```

Comment:
Trailing Newline Violation: Files should have a single trailing newline. (trailing_newline)